moveit_msgs/CollisionObject[] collision_objects
geometry_msgs/Pose pose_goal
std_msgs/Bool pick_tubule
std_msgs/Bool grab_from_top
